### ISSUE TITLE
ModalWrapper adds a div, should be a fragment?

### ISSUE THREAD

**Issue by cthurston** [dmtrKovalenko/material-ui-pickers#152]
## Steps to reproduce
1.  Create a DateTimePicker
2.  Pass classes down to the InputProps that set maxWidth, inline etc...

## Expected behavior
The input component to respect the styling.

## Actual behavior
The modal wrapper adds an extra div that I can't figure out how to get styles to.  This forces a block display of the component.

ModalWrapper.jsx
I wonder if changing the div to a fragment will fix the issue.  Otherwise we need a className or something on the div.

````
return (
      <div>
        <DateTextField
          value={value}
          format={format}
          onClick={this.togglePicker}
          // onFocus={this.togglePicker} <- Currently not properly works with .blur() on TextField
          invalidLabel={invalidLabel}
          labelFunc={labelFunc}
          {...other}
        />

        <ModalDialog
          open={this.state.open}
          onAccept={this.handleAccept}
          onDismiss={this.handleDismiss}
          dialogContentClassName={dialogContentClassName}
          okLabel={okLabel}
          cancelLabel={cancelLabel}
        >
          { children }
        </ModalDialog>
      </div>
    );
````

**Comment by alitaheri** [comment]
It's fixed on master. the next release will include the fix.

**Comment by cherniavskii** [comment]
Closed by #141
